controllers/Topics.controller.js

```diff
@@ -0,0 +1,9 @@
+const {FetchAll} = require("../models/Topics.model")
+
+exports.getTopics = ((req, res)=>{
+    FetchAll()
+    .then((topics)=>{
+        res.status(200).send(topics)
+    })
+    .catch((err)=>{res.status(404).send(err)})
```

Comment:
The error should be passed to next to be handled in app.js. Please change this line